src/check/parse/tokenize.zig

```diff
@@ -594,7 +604,7 @@ pub const Cursor = struct {
     pub fn chompIdentGeneral(self: *Cursor) void {
         while (self.pos < self.buf.len) {
             const c = self.buf[self.pos];
-            if ((c >= 'a' and c <= 'z') or (c >= 'A' and c <= 'Z') or (c >= '0' and c <= '9') or c == '_') {
+            if ((c >= 'a' and c <= 'z') or (c >= 'A' and c <= 'Z') or (c >= '0' and c <= '9') or c == '_' or c == '!') {
```

Comment:
I don't think we want to allow `!` anywhere, do we? (only at the end of names).

Reply:
Yeah, this is temporary as trying to do this at the end was frustrating me the way I tried.  I might just go back to parsing LowerIdent and then peeking for a OpBand

Reply:
This does seem reasonable to do in the tokenizer, but I think the loop here needs to be adjusted a bit. I'm fine with landing as is and fixing up later.